docs/en/stack/monitoring/how-monitoring-works.asciidoc

```diff
@@ -2,32 +2,37 @@
 [[how-monitoring-works]]
 == How monitoring works
 
-Monitoring collects data from {es} nodes, Logstash nodes, and {kib} instances.
-The {es} cluster you are monitoring controls where the monitoring metrics for
-the entire stack are stored. By default, they are stored in local indices. In
-production, we strongly recommend using a separate monitoring cluster. Using a
-separate monitoring cluster prevents production cluster outages from impacting
+Monitoring collects data from {es} nodes, Logstash nodes, and {kib} instances. 
+
+In general, the {es} cluster you are monitoring controls where the monitoring 
+metrics for the stack are stored. By default, they are stored in local indices. 
+
+In production, we strongly recommend using a separate monitoring cluster. Using 
+a separate monitoring cluster prevents production cluster outages from impacting
 your ability to access your monitoring data. It also prevents monitoring
-activities from impacting the performance of your production cluster.
+activities from impacting the performance of your production cluster. 
+
+beta[] In 6.5 and later, you can use {metricbeat} to collect and ship data about 
+{kib}, rather than routing it through {es}. The following diagram illustrates a 
+typical monitoring architecture with separate production and monitoring clusters:
+
+image::monitoring/images/architecture4.png[A typical monitoring environment]
+
+
+If you have at least a gold license, you can route data from multiple production
+clusters to a single monitoring cluster. For more information about the 
+differences between various subscription levels, see: https://www.elastic.co/subscriptions 
 
 IMPORTANT: In general, the monitoring cluster and the clusters being monitored
 should be running the same version of the stack. A monitoring cluster cannot
 monitor production clusters running newer versions of the stack. If necessary,
 the monitoring cluster can monitor production clusters running older versions,
 but the versions cannot differ by more than one major version.
 
-The following diagram illustrates a typical monitoring architecture with
-separate production and monitoring clusters:
-
-image::monitoring/images/architecture1.jpg[A typical monitoring environment]
-
-If you have at least a Gold license, you can route data from multiple production
-clusters to a single monitoring cluster:
+If you are not using {metricbeat} to collect the {kib} data, you might want to 
```

Comment:
I wouldn't say there is a causality between the usage of metricbeat and having a dedicated kibana instance. I think the previous wording is more accurate over here:

> If Kibana is a regular part of your stack, you might want to create a dedicated Kibana instance for monitoring, rather than using a single Kibana instance to access both your production cluster and monitoring cluster:

Reply:
Though when you're using Kibana with Metricbeat, you're not accessing the production cluster, are you?

Reply:
In a setup where Metricbeat is being used to monitor Kibana:
- Kibana has two connections: one to the production ES cluster and another to the Monitoring ES cluster. Of course, the two clusters might actually be the same, single cluster.
- Metricbeat has one connection: to the monitoring ES cluster.

Does this answer your question or did I just make it more confusing :)?

Reply:
Thanks, I've updated that paragraph, since it hadn't been clear to me what was meant by being "a regular part of the stack".